src/main/java/Duke.java

```diff
@@ -1,10 +1,111 @@
+import task.*;
+
+import java.io.*;
```

Comment:
Perhaps these classes could be imported explicitly? It will make your code easier to comprehend and maintain